ecc/bls12381/g1.go

```diff
@@ -2,7 +2,7 @@ package bls12381
 
 import (
 	"crypto"
-	_ "crypto/sha256" // to link library
```

Comment:
Is this (removed) comment / import still necessary? There are several other users:
```
$ git grep -nEe '_ "crypto/(sha256|sha512)'
blindsign/blindrsa/blindrsa_test.go:8:  _ "crypto/sha256"
ecc/bls12381/g1.go:5:   _ "crypto/sha256" // to link library
expander/expander_test.go:6:    _ "crypto/sha256" // to link libraries
expander/expander_test.go:7:    _ "crypto/sha512" // to link libraries
group/ristretto255.go:5:        _ "crypto/sha512" // to link libraries
hpke/kembase.go:6:      _ "crypto/sha256" // linking sha256 packages.
hpke/kembase.go:7:      _ "crypto/sha512" // linking sha512 packages.
hpke/shortkem.go:6:     _ "crypto/sha256" // linking sha256 packages.
hpke/shortkem.go:7:     _ "crypto/sha512" // linking sha512 packages.
hpke/xkem.go:6: _ "crypto/sha256" // linking sha256 packages.
hpke/xkem.go:7: _ "crypto/sha512" // linking sha512 packages.
```

Reply:
I reviewed all places where linking libraries is needed. thanks for pointing this